client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
+	},
+
+	getPrice() {
+		return ( 'bluehost' === abtest( 'signupStoreBenchmarking' ) ) ? '$3.95' : '$12.95';
+	},
+
+	render() {
+		return (
+			<div>
+				<StepHeader
+					headerText={ this.translate( 'Create a WordPress Store' ) }
+					subHeaderText={ this.translate( 'Our partners at BlueHost and WooCommerce are here for you' ) }
+				/>
+
+				<div className="design-type-with-store__container">
+					<div className="design-type-with-store__copy">
+						<img src="/calypso/images/signup/bluehost-logo.png" className="design-type-with-store__logo" />
+						{ this.translate( 'We\'ve partnered with BlueHost, a top-notch WordPress hosts with a knack for building great e-commerce stores using WooCommerce.' ) }
```

Comment:
It's kind of nice for readability in the code to break up the lines of long text, like:

```
{ this.translate( 
    'We\'ve partnered with BlueHost, a top-notch WordPress hosts ' + 
    'with a knack for building great e-commerce stores using WooCommerce.' 
) }
```